app/components/mandatarissen/fractie-selector.js

```diff
@@ -85,4 +131,38 @@ export default class MandatenbeheerFractieSelectorComponent extends Component {
     let searchResults = await this.fetchFracties(searchData);
     return searchResults;
   });
+
+  async isFractieIndependent(fractie) {
+    const type = await fractie.fractietype;
+
+    if (!type) {
+      return false;
+    }
+
+    return type.isOnafhankelijk;
+  }
+
+  async getIndependentFractie(fracties) {
+    for (const fractie of fracties) {
+      const isIndependent = await this.isFractieIndependent(fractie);
+
+      if (isIndependent) {
+        return fractie;
+      }
+    }
+    return null;
+  }
```

Comment:
Wondering why you added this? This replaces practically a oneliner, but I don't feel like this really makes it more readable, so I don't really get why. Or was it broken before?

Reply:
You are right the oneliner is better. I think while implementing I used it on multiple places thats why it moved to a spearte method, good one